Add_RO_country_fiche_2024

LauraBolocan opened the pull request.

MarcoMinghini commented:
Dear @LauraBolocan,
we see you made two pull requests, but it seems that the work is still in progress. Could you confirm? If so, please consider making one single pull request with all the edits.
Thank you, and feel free to come back to us if you have questions.

LauraBolocan commented:
Hello @MarcoMinghini 

I have deleted the other pull request and I've updated this one as the final Country Fiche. Could you tell me if it is ok?

Thank you

fabiovinci commented on RO/INIS_structure.PNG:
I removed the duplicated image.

fabiovinci commented on RO/Country_fiches/INIS_structure.PNG:
I created a dedicated folder for all the images and moved the existing one.

fabiovinci commented:
Dear @LauraBolocan

I fixed some typos, updated some links and created a dedicated folder for the images.
Everything seems ok, I will proceed with the merge.
If there's anything else that needs to be adjusted, please don't hesitate to let us know.
